test/integration/agent/flowexporter_test.go

```diff
@@ -166,6 +182,7 @@ func TestConnectionStoreAndFlowRecords(t *testing.T) {
 	testBuildFlowRecords(t, flowRecords, testConns, testConnKeys)
 }
 
+// TODO: Enhance flow exporter part of code for conditional compilation of windows
 func TestSetupConnTrackParameters(t *testing.T) {
```

Comment:
I think this test just needs to be moved to flowexporter_linux_test.go

Reply:
I added `TODO` previously because I saw this function was added by flow exporter windows support patch. If it is not for windows, then I think annotation ` +build !windows` is enough and I will remove this comment.

Reply:
Done

Reply:
I'm pretty sure the expectation is that these integration tests are run on Linux, so adding the build tag works for me

If we need to run them on both platforms, we will just move that specific test (`TestSetupConnTrackParameters`), which depends on sysctl, to a separate file